Sources/Plasma/NucleusLib/inc/plCreatableIndex.h

```diff
@@ -143,7 +143,7 @@ CLASS_INDEX_LIST_START
     CLASS_INDEX(plNetClientApp),
     CLASS_INDEX(plNetServerApp),
     CLASS_INDEX(plClient),
-    CLASS_INDEX(UNUSED_plCompoundTMModifier),
+    CLASS_INDEX(plLayerMultiply),
```

Comment:
If this class was never registered, do we actually need it?

Reply:
Good question. I'm willing to remove this commit in favor of deleting the class, if you prefer.

Reply:
I'm fine with keeping it if you think it's useful...  We'd have to adjust other tools to make use of it though, if it's never been used before.

Reply:
I'm not sure it is useful... In a modern, shader based system these layers are just a legacy data source. :/